src/ProjectNeon/Assets/Data/Templates/Editor/GameDataMenu.cs

```diff
@@ -18,6 +18,15 @@ static class GameDataMenu
     [MenuItem("Assets/Create/GameContent/DumbAI")]
     static void DumbAI() => Create<DumbAI>();
 
+    [MenuItem("Assets/Create/Variable/Spell Hits Last Card Multiplier")]

```

Comment:
Don't leave editor menu items for things that will only be created once. 

Just create one and then kill the editor code.